atd-vze/src/views/Locations/locationGridTableParameters.js

```diff
@@ -0,0 +1,32 @@
+export const locationsGridTableAdvancedFilters = {
```

Comment:
copied from the crashes grid table filters. TBH i don't expect anyone to use these, but the "invert_toggle_state" setting has the affect of causing these filters to be applied by default, which is what we want.

Reply:
this makes sense to me! retain the ability to nav to any location in the database but also feature those of most interest in the list view 👍